Fix antiaim

fartdev opened the pull request:
Freelook with antiaim disabled remains unchanged but this pr fixes the freelook fix, so now you can walk around with antiaim on

fartdev committed:
Fix antiaim

fartdev committed:
buil

0x3C50 commented on src/main/java/coffee/client/mixin/entity/LivingEntityMixin.java:
```diff
@@ -69,7 +69,7 @@ public void coffee_cancelCollision(Entity entity, CallbackInfo ci) {
     @Redirect(method = "jump", at = @At(value = "INVOKE", target = "net/minecraft/entity/LivingEntity.getYaw()F"), require = 0)
     private float coffee_overwriteJumpDirection(LivingEntity instance) {
-        if (instance.equals(CoffeeMain.client.player) && ModuleRegistry.getByClass(FreeLook.class).isEnabled()) {
+        if (instance.equals(CoffeeMain.client.player) && ModuleRegistry.getByClass(FreeLook.class).isEnabled() && !((boolean) ModuleRegistry.getByClass(FreeLook.class).config.get("Enable Anti-Aim").getValue())) {
```
Change to getter for direct reference, getting by name is wonky as shit

0x3C50 commented on src/main/java/coffee/client/mixin/render/GameRendererMixin.java:
```diff
@@ -92,7 +92,7 @@ private void coffee_stopCursorBob(MatrixStack matrices, float f, CallbackInfo ci
     @Redirect(at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/Entity;raycast(DFZ)Lnet/minecraft/util/hit/HitResult;"), method = "updateTargetedEntity", require = 0)
     HitResult coffee_replaceHitResult(Entity instance, double maxDistance, float tickDelta, boolean includeFluids) {
-        if (ModuleRegistry.getByClass(FreeLook.class).isEnabled()) {
+        if (ModuleRegistry.getByClass(FreeLook.class).isEnabled() && !((boolean) ModuleRegistry.getByClass(FreeLook.class).config.get("Enable Anti-Aim").getValue())) {
```
See above

0x3C50 reviewed (changes requested):
Good commit but needs changes

fartdev committed:
This should be better?

I looked through the codebase and this is how you access AntiCrash config for EntityRendererMixin

fartdev committed:
Build

Version/changelog unchanged as it is the same version as the previous fix

fartdev commented:
Should be better now, I used the same method you used for AntiCrash
